hack/build/common.sh

```diff
@@ -16,6 +16,15 @@ CDI_DIR="$(readlink -f $(dirname $0)/../../)"
 BIN_DIR=${CDI_DIR}/bin
 OUT_DIR=${CDI_DIR}/_out
 CMD_OUT_DIR=${OUT_DIR}/cmd
+TESTS_OUT_DIR=${OUT_DIR}/tests
 BUILD_DIR=${CDI_DIR}/hack/build
 MANIFEST_TEMPLATE_DIR=${CDI_DIR}/manifests/templates
 MANIFEST_GENERATED_DIR=${CDI_DIR}/manifests/generated
+
+function build_func_tests() {
```

Comment:
Is this file the right place for this func?  It looks like it's only used by one script.